src/db/tables/qa_customs.ts

```diff
@@ -111,23 +114,35 @@ async function getQaCustomBrokers(
  * @param brokerData This object conforms to QaBrokerData, which is the same as
  * OnerepScanResulsRow with some fields omitted due to them being automaticallty set.
  */
-async function addQaCustomBroker(brokerData: QaBrokerData): Promise<void> {
-  await knex("qa_custom_brokers").insert({
-    ...brokerData,
-    emails: JSON.stringify(brokerData.emails),
-    phones: JSON.stringify(brokerData.phones),
-    addresses: JSON.stringify(brokerData.addresses),
-    relatives: JSON.stringify(brokerData.relatives),
-  });
-  logger.info(`Created a custom broker: ${brokerData.data_broker}`);
+async function addQaCustomBroker(
+  brokerData: OnerepScanResultDataBrokerRow,
+): Promise<void> {
+  try {
+    await knex("qa_custom_brokers").insert({
+      ...brokerData,
+      emails: JSON.stringify(brokerData.emails),
+      phones: JSON.stringify(brokerData.phones),
+      addresses: JSON.stringify(brokerData.addresses),
+      relatives: JSON.stringify(brokerData.relatives),
+    });
+
+    logger.info(`Created a custom broker: ${brokerData.data_broker}`);
+  } catch (error) {
+    logger.error(`Error creating custom broker: ${brokerData.data_broker}`);
+    logger.error(`Error message: ${error}`);
+    logger.error(`Error stack: ${error}`);
+    throw new Error(
+      `Failed to insert broker data for ${brokerData.data_broker}`,
+    );
+  }
 }
 
-async function getAllQaCustomBrokers(
-  onerep_profile_id: number,
-): Promise<QaBrokerData[]> {
-  const res = (await knex("qa_custom_brokers")
-    .where("onerep_profile_id", onerep_profile_id)
-    .select("*")) as QaBrokerData[];
+async function getAllQaCustomBrokers(): Promise<
```

Comment:
I think we should rename this to something like `scanResultWithBrokers`.. also for the UI, it might be confusing to see the word "brokers" and see `scan results` attributes